api/app/models/spree/api/webhooks/order.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Api
+    module Webhooks
+      module Order
+        def cancel
+          super
+          Spree::Webhooks::Endpoints::QueueRequests.call(event: 'order.cancel', payload: {})
+        end
+
+        def finalize!
+          super
```

Comment:
@sebastian-palma and here

Reply:
This is the only exception @damianlegawiec, `Order#finalize!` is the `after_transition` method to run after invoking `Order#complete`.